lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl

```diff
@@ -644,23 +646,34 @@
   <xsl:preserve-space elements="ltx:date"/>
   <xsl:template match="ltx:date"/>
 
-  <xsl:template match="ltx:date" mode="intitle">
+  <xsl:template name="dates">
     <xsl:param name="context"/>
-    <xsl:text>&#x0A;</xsl:text>
-    <!-- Originally, html5 seemed to suggest we might use h2 here, but that is retracted-->
-    <xsl:element name="div" namespace="{$html_ns}">
-      <xsl:call-template name="add_id"/>
-      <xsl:call-template name="add_attributes"/>
-      <xsl:apply-templates select="." mode="begin">
-        <xsl:with-param name="context" select="$context"/>
-      </xsl:apply-templates>
-      <xsl:apply-templates select="//ltx:document/ltx:date/node()">
-        <xsl:with-param name="context" select="$context"/>
-      </xsl:apply-templates>
-      <xsl:apply-templates select="." mode="end">
-        <xsl:with-param name="context" select="$context"/>
-      </xsl:apply-templates>
-    </xsl:element>
+    <xsl:param name="dates" select="ltx:date"/>
+    <xsl:if test="$dates">
+      <xsl:text>&#x0A;</xsl:text>
+      <!-- Originally, html5 seemed to suggest we might use h2 here, but that is retracted-->
+      <xsl:element name="div" namespace="{$html_ns}">
+        <xsl:attribute name="class">ltx_dates</xsl:attribute>
+        <xsl:apply-templates select="." mode="begin">
+          <xsl:with-param name="context" select="$context"/>
+        </xsl:apply-templates>
+        <xsl:apply-templates select="$dates" mode="inintitle">
+          <xsl:with-param name="context" select="$context"/>
+        </xsl:apply-templates>
+        <xsl:apply-templates select="." mode="end">
+          <xsl:with-param name="context" select="$context"/>
+        </xsl:apply-templates>
+      </xsl:element>
+    </xsl:if>
+  </xsl:template>
+
+  <xsl:template match="ltx:date" mode="inintitle">
+    <xsl:param name="context"/>
+    <xsl:if test="@name"><xsl:value-of select="@name"/><xsl:text>: </xsl:text></xsl:if>
```

Comment:
This is a cool approach. I tested locally and noticed that `@name` actually has no attribute to select against in the aastex63 example. The attribute for `ltx:date` that is currently in use is role, and switching this test to `@role` got me the visuals I expected locally. Was there a reason for `@name`?